modules/transports/optional/fix/src/test/java/org/apache/synapse/transport/fix/FIXTransportSenderInitTest.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.synapse.transport.fix;
+
+import junit.framework.TestCase;
+import org.apache.axis2.context.ConfigurationContext;
+import org.apache.axis2.description.AxisService;
+import org.apache.axis2.description.Parameter;
+import org.apache.axis2.description.TransportOutDescription;
+import org.apache.axis2.engine.AxisConfiguration;
+import org.apache.log4j.BasicConfigurator;
+import org.testng.annotations.Test;
+
+public class FIXTransportSenderInitTest extends TestCase {
+    @Test()
+    public void testFIXTransportSenderInit() throws Exception {
```

Comment:
move the test method to FIXTransportListenerTest class

Reply:
done